gatsby-config.ts

```diff
@@ -1,63 +1,100 @@
-import type { GatsbyConfig, PluginRef } from "gatsby"
-import "dotenv/config"
+import type { GatsbyConfig, PluginRef } from 'gatsby';
+import 'dotenv/config';
 
-const shouldAnalyseBundle = process.env.ANALYSE_BUNDLE
+const shouldAnalyseBundle = process.env.ANALYSE_BUNDLE;
+const siteUrl =
+  process.env.NODE_ENV == 'development'
+    ? 'http://localhost:8000/'
+    : 'https://romanrud.com/';
 
 const config: GatsbyConfig = {
   siteMetadata: {
     // You can overwrite values here that are used for the SEO component
     // You can also add new values here to query them like usual
     // See all options: https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-minimal-blog/gatsby-config.mjs
-      // Used for the title template on pages other than the index site
-      siteTitle: `Roman Rud Blog`,
-      // Default title of the page
-      siteTitleAlt: `Roman Rud Blog`,
-      // Can be used for e.g. JSONLD
-      siteHeadline: `Roman Rud Blog`,
-      // Will be used to generate absolute URLs for og:image etc.
-      siteUrl: `https://romanrud.com`,
-      // Used for SEO
-      siteDescription: `Roman Rud Personal Blog`,
-      // Will be set on the <html /> tag
-      siteLanguage: `ru`,
-      // Used for og:image and must be placed inside the `static` folder
-      siteImage: `/R.jpeg`,
-      // Twitter Handle
-      author: `Roman Rud`,
+    // Used for the title template on pages other than the index site
+    siteTitle: 'Roman Rud Blog',
+    // Default title of the page
+    siteTitleAlt: 'Roman Rud Blog',
+    // Can be used for e.g. JSONLD
+    siteHeadline: 'Roman Rud Blog',
+    // Will be used to generate absolute URLs for og:image etc.
+    siteUrl: 'https://romanrud.com',
+    // Used for SEO
+    siteDescription: 'Roman Rud Personal Blog',
+    // Will be set on the <html /> tag
+    siteLanguage: 'en',
+    // Used for og:image and must be placed inside the 'static' folder
+    siteImage: '/R.jpeg',
+    // Twitter Handle
+    author: 'Roman Rud',
   },
-  trailingSlash: `never`,
+  trailingSlash: 'always',
   plugins: [
     {
-      resolve: `@lekoarts/gatsby-theme-minimal-blog`,
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        path: `${__dirname}/locales`,
+        name: 'locale',
+      },
+    },
+    {
+      resolve: 'gatsby-plugin-react-i18next',
+      options: {
+        localeJsonSourceName: 'locale',
+        languages: ['uk', 'ru', 'en'], // Add your supported languages here
```

Comment:
@kaleemullahdev 'uk' looks really weird. Could you please try again to make it work for 'ua'? Since it worked fine under the /ua/ path.